lnbits/core/services.py

```diff
@@ -382,3 +382,28 @@ async def check_transaction_status(
 # WARN: this same value must be used for balance check and passed to WALLET.pay_invoice(), it may cause a vulnerability if the values differ
 def fee_reserve(amount_msat: int) -> int:
     return max(int(RESERVE_FEE_MIN), int(amount_msat * RESERVE_FEE_PERCENT / 100.0))
+
+
+class websocketConnectionManager:
+    def __init__(self):
+        self.active_connections: List[WebSocket] = []
+
+    async def connect(self, websocket: WebSocket, item_id: str):
+        await websocket.accept()
```

Comment:
- should be checked if the `item_id` is already in `active_connections`. 
    - avoid appending multiple entries for the same `item_id`